tests/e2e/cucumber/features/integrations/reshare.ocis.feature

```diff
@@ -0,0 +1,46 @@
+Feature: reshare
+
+  Background:
+    And "Admin" disables share auto accepting
+
+  Scenario: folder
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+      | Brian |
+      | Carol |
+    And "Admin" creates following group
+      | id      |
+      | sales   |
+      | finance |
+    When "Admin" adds user to the group
+      | user  | group |
+      | Carol | sales |
+    When "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" navigates to the personal space page
+    And "Alice" creates the following resources
+      | resource         | type   |
+      | folder_to_shared | folder |
+    When "Alice" shares the following resource using the sidebar panel
+      | resource         | user  | role   |
+      | folder_to_shared | Brian | editor |
+    And "Brian" logs in
+    And "Brian" opens the "files" app
+    And "Brian" navigates to the shared with me page
+    And "Brian" accepts the following share
+      | name             |
+      | folder_to_shared |
+    And "Brian" reshares the following resource
+      | resource         | user  | role   |
+      | folder_to_shared | Carol | viewer |
+    When "Carol" logs in
+    And "Carol" opens the "files" app
+    And "Carol" navigates to the shared with me page
+    And "Carol" accepts the following share
+      | name             |
+      | folder_to_shared |
+    And "Carol" logs out
```

Comment:
```suggestion
    Then the following users should be displayed in the collaborator list of folder "folder_to_shared" for "Alice"
    |user|role|
    |Brian|editor|
    |Carol|viewer|
    And "Carol" logs out
```
Can we add a step like this here?

Reply:
We had already used the 'then' steps, but then removed them (not my decision)
`Then the following users should be displayed in the collaborator list of folder "folder_to_shared" for "Alice"` steps partly checked in `And "Carol" accepts the following share` 
I sticked to this style